include/paths.h

```diff
@@ -37,8 +37,8 @@
 
 #include <sys/cdefs.h>
 
-#ifdef COMPAT_64BIT
-#define	_PATH_LOCALBASE	"/usr/local64"
+#ifdef COMPAT_libcompat
```

Comment:
Hmmm, this has subtle changes for COMPAT32 that aren't present upstream.  I think we also have decided against having /usr/local64c on hybrid worlds?

Reply:
It does, but we will at least want a local64cb in time, and I don't really see what harm it does for lib32? As for hybrid, we don't support ports or packages there at all so I don't personally care what happens.

Reply:
But if desired I can just change this to
```
#if defined(COMPAT_LIB64)
#define _PATH_LOCALBASE "/usr/local" COMPAT_libcompat
```
and then later add `|| defined(COMPAT_LIB64CB)`

Reply:
I think I might prefer we do that.

Reply:
FWIW, I suspect upstream will find they want /usr/local32 at some point after we kill off 32-worlds, but I don't think I'm in a rush to push that through